data/working_groups.yml

```diff
@@ -0,0 +1,74 @@
+---
+# template
+# - title:
+#   goal:
+#   open_letter:
+#   communication_channel:
+#   url:
+#   chairs:
+#     - name:
+#       email:
+#     - name:
+#       email:
+#   status:
+#   outcome:
+#       - url:
+#       - url:
+#
+
+# ACTIVE
+
+-   title: BIDS 2.0
+    goal: Discussions and suggestions of backwards incompatible changes to BIDS
+    open_letter: TBD
+    communication_channel: TBD
+    url: https://github.com/bids-standard/bids-2-devel
+    chairs:
+    -   name: Yaroslav Halchenko
+        email: debian@onerussian.com
+    status: active
+
+-   title: Governance amendments
+    goal: Clarification of the BEP process
+    open_letter: TBD
+    communication_channel: TBD
+    url:
```

Comment:
@arokem  Can you give me some links for the open letter, a communication channel where people can join and interact with the working group and URL that points to where the work is happening

Reply:
updated